### DISCUSSION TITLE
When will a update come out?

### DISCUSSION THREAD

**Discussion by Minecraftmodster** [jebbyk/OSBES-minecraft-bedrock-edition-shader#166] [Q&A]
When will the 1.13.0 update of the osbes shader come out?

**Comment by AverageCrafter** [comment]
Someone asked that 10 days ago, and still no answer

**Reply by Minecraftmodster** [reply]
> Someone asked that 10 days ago, and still no answer

Oh okay

**Comment by hope3x** [comment]
There's no specific date for the new release, but expect it to get released soon.

**Comment by OpenSauce04** [comment]
It will likely release relatively soon if no complications come up
